man/ctags.1.rst.in

```diff
@@ -20,54 +20,174 @@ DESCRIPTION
+Alternatively, @CTAGS_NAME_EXECUTABLE@ can generate a cross reference
 file which lists, in human readable form, information about the various
 source objects found in a set of language files.
 
 Tag index files are supported by numerous editors, which allow the user to
 locate the object associated with a name appearing in a source file and
-jump to the file and line which defines the name. Those known about at
-the time of this release are:
+jump to the file and line which defines the name. See the manual of your
+favorite editor about utilizing @CTAGS_NAME_EXECUTABLE@ command and
+the tag index files in the editor.
 
-	**Vi** (1) and its derivatives (e.g. Elvis, Vim, Vile, Lemmy), **CRiSP**,
-	**Emacs**, **FTE** (Folding Text Editor), **JED**, **jEdit**, **Mined**,
-	**NEdit** (Nirvana Edit), **TSE** (The SemWare Editor), **UltraEdit**,
-	**WorkSpace**, **X2**, **Zeus**
-
-**@CTAGS_NAME_EXECUTABLE@** is capable of generating different kinds of tags
+@CTAGS_NAME_EXECUTABLE@ is capable of generating different kinds of tags
 for each of many different languages. For a complete list of supported
 languages, the names by which they are recognized, and the kinds of tags
 which are generated for each, see the ``--list-languages`` and ``--list-kinds``
 options.
 
+This man page describes Universal-ctags, an implementation of ctags
+derived from Exuberant-ctags. The major incompatible changes between
+Universal-ctags and Exuberant-ctags are enumerated in
+ctags-incompatibilities(1).
+
+One of the advantages of Exuberant-ctags is that it allows an user to
+define a new parser from command line. Extending this feature is one
+of the major topics in Universal-ctags development. ctags-optlib(1)
+describes how the feature is extended.
+
+Newly introduced premature features are not explained here. If you
+are interested in such features and ctags internal,
+visit http://docs.ctags.io/en/latest/.
+
 
 SOURCE FILES
 ------------
 
 Unless the ``--language-force`` option is specified, the language of each source
 file is automatically selected based upon a mapping of file names to
-languages. The mappings in effect for each language may be display using
+languages. The mappings in effect for each language may be displayed using
 the ``--list-maps`` option and may be changed using the ``--langmap`` option. On
 platforms which support it, if the name of a file is not mapped to a
-language and the file is executable, the first line of the file is checked
-to see if the file is a "#!" script for a recognized language.
-
-By default, all other files names are ignored. This permits running
-@CTAGS_NAME_EXECUTABLE@ on all files in either a single directory (e.g.
-"@CTAGS_NAME_EXECUTABLE@ \*"), or on all files in an entire source directory
-tree (e.g. "@CTAGS_NAME_EXECUTABLE@ -R"), since only those files whose
+language, @CTAGS_NAME_EXECUTABLE@ tries to guess the language for
+the file by inspecting its content. See "GUESSING PARSER".
```

Comment:
"GUESSING PARSER" -> "Guessing parser"
Cases are not matched with the following subsection name.